pilot/pkg/serviceregistry/serviceentry/servicediscovery.go

```diff
@@ -360,7 +361,7 @@ func (s *ServiceEntryStore) serviceEntryHandler(_, curr config.Config, event mod
 	fullPush := len(configsUpdated) > 0
 	// if not full push needed, at least one service unchanged
 	if !fullPush {
-		s.edsUpdate(serviceInstances, true)
+		s.queueEdsPush(serviceInstances)
```

Comment:
could we call `edsUpdate` and `edsCacheUpdate`? 

queueEdsPush can not express this is a doing cache update -> push in searial

Reply:
Sorry. I did not understand. We already have edsUpate and edsCacheUpdate that call xds update functions. What are you suggesting here?

Reply:
I mean keep the name `edsUpdate` and `edsCacheUpdate`

Reply:
Changed a bit. PTAL